lib/elixir/pages/anti-patterns/process-anti-patterns.md

```diff
@@ -4,12 +4,216 @@ This document outlines anti-patterns related to processes and process-based abst
 
 ## Code organization by process
 
-TODO.
+* **Problem:** This anti-pattern refers to code that is unnecessarily organized by processes. A process itself does not represent an anti-pattern, but it should only be used to model runtime properties (e.g., concurrency, access to shared resources, and event scheduling). When a process is used for code organization, it can create bottlenecks in the system.
```

Comment:
```suggestion
* **Problem:** This anti-pattern refers to code that is unnecessarily organized by processes. A process itself does not represent an anti-pattern, but it should only be used to model runtime properties (e.g., concurrency, access to shared resources, and event scheduling). However, when a process is used for code organization, it can create bottlenecks in the system.
```